.rubocop.yml

```diff
@@ -55,3 +52,18 @@ Style/StringLiterals:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+Style/StringLiterals:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/MethodLength:
+  Enabled: false
+
+Lint/AssignmentInCondition:
+  Enabled: false
```

Comment:
My _personal_ opinion is "yes" for `while` and "no" for `if`, but that doesn't really seem fair to push on people, so turning it off is ok for me!